### PR TITLE
add practical tasks on feature branch

### PR DESCRIPTION
**Pull request by Levan787**

### CODE REVIEW

File: Objects-and-Functions/pears-apples.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+write a function that returns the result of adding the number of accepted pears and apples
+ to the number of accepted apples
+*/ 
+
+
+function warehouse(pears, apples) {
+  return pears + apples;
+}
+let pears = Number(prompt("How many pears do you want to add?"));
+let apples = Number(prompt("How many apples do you want to add?"));
```
Comment (rimus) [inline Objects-and-Functions/pears-apples.js:11]: Please don't use prompts and alerts in your code. Log to console for visual representation and add values directly in your code (e.g. 
```
let pears = 10;
let apples = 3;
```

File: Objects-and-Functions/pears-apples.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+let result = warehouse(pears, apples);
+alert(result);
```
Comment (rimus) [inline Objects-and-Functions/pears-apples.js:13]: Please don't use prompts and alerts in your code. Log to console for visual representation and add values directly in your code.

File: Objects-and-Functions/prime-or-not.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+        return 'This is not prime number';
+    }
+    for (let i = 2; i < number; i++) {
+        if (number % i === 0) {
+            return 'This is not prime number';
+        }
+    }
+    return 'This is prime number';
+}
+let  prime_number = prompt("Enter a prime number");
```
Comment (rimus) [inline Objects-and-Functions/prime-or-not.js:14]: Please don't use prompts and alerts in your code. Log to console for visual representation and add values directly in your code.

File: Objects-and-Functions/prime-or-not.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+let result = isPrime(prime_number);
+alert(result);
```
Comment (rimus) [inline Objects-and-Functions/prime-or-not.js:16]: Please don't use prompts and alerts in your code. Log to console for visual representation and add values directly in your code.

File: Objects-and-Functions/terminal.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
+ /* -  Your name is saved in the payment terminal,
+  write a function to define the name in the terminal 
+  if you entered your name, then hello + name, if not, then there is no such name
+  */
+  function payment(name) {
+    if (name === '') {
+        return 'There is no such name';
```
Comment (rimus) [inline Objects-and-Functions/terminal.js:7]: Not quite right: the function must compare the passed name to some specific value.

File: Objects-and-Functions/terminal.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+    } else {
+        return 'Hello ' + name;
+    }
+}
+let name2 = prompt("Enter your name");
```
Comment (rimus) [inline Objects-and-Functions/terminal.js:12]: Please don't use prompts and alerts in your code. Log to console for visual representation and add values directly in your code.

File: Objects-and-Functions/terminal.js [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+let result = payment(name2);
+alert(result);
```
Comment (rimus) [inline Objects-and-Functions/terminal.js:14]: Please don't use prompts and alerts in your code. Log to console for visual representation and add values directly in your code.

File: Objects-and-Functions/type-argument.js [hunk 8]
```diff
@@ -0,0 +1,29 @@
+
+    
+    //  Write a function for calculating the type of arguments and output to console
+    function calculateType(...args) {
+        for (let i = 0; i < args.length; i++) {
+            console.log(typeof args[i]);
+        }
+    }
+    console.log(calculateType(prompt("Enter a type of arguments")));
```
Comment (rimus) [inline Objects-and-Functions/type-argument.js:9]: Please don't use prompts and alerts in your code. Log to console for visual representation and add values directly in your code.

File: Arrays-cycles/10odd-numbers-loop.js [hunk 9]
```diff
@@ -0,0 +1,11 @@
+
+//10)
+//Implement a loop that prints odd numbers to the console.
+
+let number = 0;
+
+while (true) {
+    if (number % 2 != 0)
```
Comment (rimus) [inline Arrays-cycles/10odd-numbers-loop.js:8]: `!==`

File: Arrays-cycles/1movies-name.js [hunk 10]
```diff
@@ -0,0 +1,22 @@
...
+// 1)
+// Given an array consisting of movie names, 
+//iterate over the array with the output of the names of each movie to the console. 
+//For example, if the array is ["The Matrix", "Star Wars", "The Shawshank Redemption"]
+//the output should be:
+//The Matrix
+//Star Wars
+//The Shawshank Redemption
+var movies = ["The Matrix", "Star Wars", "The Shawshank Redemption"]; //create an array of movies
+for (var i = 0; i < movies.length; i++) { //iterate over the array
```
Comment (rimus) [inline Arrays-cycles/1movies-name.js:11]: No `var`. Please use `const` or `let`.

File: Arrays-cycles/1movies-name.js [hunk 11]
```diff
@@ -0,0 +1,22 @@
+
+// 1)
+// Given an array consisting of movie names, 
+//iterate over the array with the output of the names of each movie to the console. 
+//For example, if the array is ["The Matrix", "Star Wars", "The Shawshank Redemption"]
+//the output should be:
+//The Matrix
+//Star Wars
+//The Shawshank Redemption
+var movies = ["The Matrix", "Star Wars", "The Shawshank Redemption"]; //create an array of movies
```
Comment (rimus) [inline Arrays-cycles/1movies-name.js:10]: No `var`. Please use `const` or `let`.

File: Arrays-cycles/2car-manufacturers.js [hunk 12]
```diff
@@ -0,0 +1,9 @@
+// 2)
+// given an array of car manufacturers, convert the array to a string and back to an array. 
+//For example, if the array is ["Ford", "Chevy", "Dodge", "Honda"]
+//the output should be:
+//["Ford", "Chevy", "Dodge", "Honda"]
+var carManufacturers = ["Ford", "Chevy", "Dodge", "Honda"]; //create an array of car manufacturers
+var carManufacturersString = carManufacturers.toString(); //convert the array to a string
+var carManufacturersArray = carManufacturersString.split(","); //convert the string to an array
```
Comment (rimus) [inline Arrays-cycles/2car-manufacturers.js:8]: No `var`. Please use `const` or `let`.

File: Arrays-cycles/3friends-name.js [hunk 13]
```diff
@@ -0,0 +1,10 @@
+// 3)
+//Given an array of the names of your friends, add the words hello to each element of the array.
+//For example, if the array is ["John", "Jane", "Mary"]
+//the output should be:
+//["Hello John", "Hello Jane", "Hello Mary"]
+var friends = ["John", "Jane", "Mary"]; //create an array of friends
+for (var i = 0; i < friends.length; i++) { //iterate over the array
```
Comment (rimus) [inline Arrays-cycles/3friends-name.js:7]: No `var`. Please use `const` or `let`.

File: Arrays-cycles/4numeric-boolean.js [hunk 14]
```diff
@@ -0,0 +1,10 @@
+//4)
+//Convert numeric array to Boolean.
+//For example, if the array is [1, 0, 1, 1]
+//the output should be:
+//[true, false, true, true]
+var numbers = [1, 0, 1, 1]; //create an array of numbers
+for (var i = 0; i < numbers.length; i++) { //iterate over the array
```
Comment (rimus) [inline Arrays-cycles/4numeric-boolean.js:7]: No `var`. Please use `const` or `let`.

File: Arrays-cycles/5sort-array.js [hunk 15]
```diff
@@ -0,0 +1,10 @@
+//5)
+//Sort the array [1,6,7,8,3,4,5,6] in descending order.
+//the output should be:
+//[8,7,6,6,5,4,3,1]
+var numbers = [1, 6, 7, 8, 3, 4, 5, 6]; //create an array of numbers
+numbers.sort(function(a, b) { //sort the array in descending order
```
Comment (rimus) [inline Arrays-cycles/5sort-array.js:6]: No `var`. Please use `const` or `let`.

File: Arrays-cycles/6filter-array.js [hunk 16]
```diff
@@ -0,0 +1,10 @@
+//6)
+//Filter array [1,6,7,8,3,4,5,6] by value> 3.
+//the output should be:
+//[6,7,8,6]
+var numbers = [1, 6, 7, 8, 3, 4, 5, 6]; //create an array of numbers
+var filteredNumbers = numbers.filter(function(number) { //filter the array by value > 3
```
Comment (rimus) [inline Arrays-cycles/6filter-array.js:6]: No `var`. Please use `const` or `let`.

File: Arrays-cycles/7two-parametres.js [hunk 17]
```diff
@@ -0,0 +1,7 @@
+//7)
+//Write a function that takes two parameters - an array and a number and outputs the index of an array element equal to a number.
+//For example, if the array is [1,6,7,8,3,4,5,6] and the number is 8, the output should be:
+//3
+var numbers = [1, 6, 7, 8, 3, 4, 5, 6]; //create an array of numbers
+var index = numbers.indexOf(7); //find the index of the element equal to 6
```
Comment (rimus) [inline Arrays-cycles/7two-parametres.js:6]: No `var`. Please use `const` or `let`.

File: Arrays-cycles/9prime-numbers.js [hunk 18]
```diff
@@ -0,0 +1,30 @@
+//9)
+//Implement a loop that prints prime numbers to the console.
+// JavaScript program to display Prime numbers till N
+// function to check if a given number is prime
+function isPrime( n)
+{
+	// since 0 and 1 is not prime return false.
+	if(n == 1 || n == 0) return false;
```
Comment (rimus) [inline Arrays-cycles/9prime-numbers.js:8]: `===`

File: Arrays-cycles/9prime-numbers.js [hunk 19]
```diff
@@ -0,0 +1,30 @@
...
+
+	// Run a loop from 2 to n-1
+	for(var i = 2; i < n; i++)
```
Comment (rimus) [inline Arrays-cycles/9prime-numbers.js:11]: No var. Please use const or let.

File: Arrays-cycles/9prime-numbers.js [hunk 20]
```diff
@@ -0,0 +1,30 @@
...
+	{
+	
+		// if the number is divisible by i, then n is not a prime number.
+		if(n % i == 0) return false;
```
Comment (rimus) [inline Arrays-cycles/9prime-numbers.js:15]: `===`

File: Arrays-cycles/9prime-numbers.js [hunk 21]
```diff
@@ -0,0 +1,30 @@
...
+	}
+	// otherwise, n is prime number.
+	return true;
+}
+// Driver code
+var N = Number(prompt("write number"));
```
Comment (rimus) [inline Arrays-cycles/9prime-numbers.js:21]: 1. No `var`. Please use `const` or `let`.
2. Please don't use prompts and alerts in your code. Log to console for visual representation and add values directly in your code.
3. Naming: variables should not start with capitals.

File: Arrays-cycles/9prime-numbers.js [hunk 22]
```diff
@@ -0,0 +1,30 @@
...
+
+// check for every number from 1 to N
+for(var i = 1; i <= N; i++)
```
Comment (rimus) [inline Arrays-cycles/9prime-numbers.js:24]: No `var`. Please use `const` or `let`.

File: exception-handling/Exception-handling.js [hunk 23]
```diff
@@ -0,0 +1,28 @@
+//In a try catch construction, wrap the code: console.log (a), let a = 3. 
+//And display an error - ‘let must be declared’ before use. 
+//When running 1/0, the error 'cannot be divided by zero' is displayed.
```
Comment (rimus) [inline exception-handling/Exception-handling.js:3]: Haven't seen solution for this part.

File: string-regexp-datetime/date.js [hunk 24]
```diff
@@ -0,0 +1,10 @@
+// Get the day, month and year of the current date and output it to the console separately.
+
+
+var date = new Date();
+var day = date.getDate();
+var month = date.getMonth() + 1;
+var year = date.getFullYear();
```
Comment (rimus) [inline string-regexp-datetime/date.js:7]: No `var`. Please use `const` or `let`.

File: string-regexp-datetime/2regex.js [hunk 25]
```diff
@@ -0,0 +1,8 @@
+//Given the string '2 + 3 223 2223'. 
+//Write a regex that finds line 2 + 3 without capturing the rest.
+
+
+var str = '2 + 3 223 2223';
+var regex = /2\s\+\s3/g;
+var result = str.match(regex);
```
Comment (rimus) [inline string-regexp-datetime/2regex.js:7]: No `var`. Please use `const` or `let`.

File: string-regexp-datetime/1regex.js [hunk 26]
```diff
@@ -0,0 +1,14 @@
+//1)Given the string 'ahb acb aeb aeeb adcb axeb'. 
+//Write a regular expression that matches the strings ahb, acb, aeb by pattern: letter 'a', any character, letter 'b'.
+
+
+var str = 'ahb acb aeb aeeb adcb axeb';
+var regex = /a[^b]b/g;
+var result = str.match(regex);
```
Comment (rimus) [inline string-regexp-datetime/1regex.js:7]: No `var`. Please use `const` or `let`.

**Review by rimus — COMMENTED** [review]
Need to fix comments.

**Comment by Levan787** [comment]
okay i will adjust it

**Pull request merged**